testsuite/ext-tools/maintenance_json_generator.py

```diff
@@ -77,10 +77,12 @@
 nodesdict42 = {
     "server": ["/SUSE_Updates_SLE-Module-SUSE-Manager-Server_4.2_x86_64/",
                "/SUSE_Updates_SLE-Product-SUSE-Manager-Server_4.2_x86_64/",
+               "SUSE_Updates_SLE-Product-SLES_15-SP3-LTSS_x86_64/",
                "/SUSE_Updates_SLE-Module-Basesystem_15-SP3_x86_64/",
                "/SUSE_Updates_SLE-Module-Server-Applications_15-SP3_x86_64/"],
     "proxy": ["/SUSE_Updates_SLE-Module-SUSE-Manager-Proxy_4.2_x86_64/",
               "/SUSE_Updates_SLE-Product-SUSE-Manager-Proxy_4.2_x86_64",
+               "SUSE_Updates_SLE-Product-SLES_15-SP3-LTSS_x86_64/",
```

Comment:
I wonder if we need those 4.2 changes anymore, because of its EOL after Oct. 31th?

Reply:
maybe not but we can be using the script for sle updates even later maybe. So not sure. I would like to include it just to be safe. If we remove it later we can remove it whole.